lms/djangoapps/static_template_view/tests/test_views.py

```diff
@@ -26,6 +37,20 @@ def test_about(self):
         """
         self._test_view('about', 'text/html')
 
+    def test_about_with_site_configuration(self):
+        """
+        Test the about view with the header and content set in SiteConfiguration.
+        """
+        test_header = "Very Unique Test Header"
+        test_content = "Very Unique Test Content"
+        test_header_key = 'static_template_about_header'
+        test_content_key = 'static_template_about_content'
+        response = None
+        with with_site_configuration_context(configuration={test_header_key: test_header,test_content_key: test_content}):
```

Comment:
You still have a PEP8 violation in this line (which causes the quality Jenkins task to fail): missing space after the comma.